src/db.py

```diff
@@ -53,15 +53,18 @@ def get_course(cid):
 	return mongo.db.courses.find_one({"_id": cid})
 
 
-def get_assignments_for_course(cid):
-	return list(mongo.db.assignments.find({"course_id": cid}))
+def get_assignments_for_course(cid, visible=True):
```

Comment:
The optional argument here is named slightly confusingly. If `visible=False`, intuitively, it would mean all assignments whose visibility is False, but this returns all assignments.
I think replacing `visible` to `visible_only` makes more sense.